modules/nw-metallb-incompat-ipfailover.adoc

```diff
@@ -0,0 +1,43 @@
+[id="nw-metallb-incompat-failover_{context}"]
+= MetalLB incompatibility with IP failover and Keepalived
+
+The {product-title} 4.8 release added IP failover for pods and services.
+IP failover is implemented with Keepalived.
+
+IP failover and MetalLB are similar in that they both provide fault tolerance for IP addresses that are external to the cluster.
+However, Keepalived and MetalLB are incompatible.
+If you enabled IP failover on your cluster, you can disable it and remove Keepalived before you start MetalLB on your cluster.
```

Comment:
do we want to be more specific and undo everthing that was done by https://deploy-preview-34728--osdocs.netlify.app/openshift-enterprise/latest/networking/configuring-ipfailover.html#configuring-ipfailover, then we cleanup with the following yaml ?